deployment/scripts/cluster.backup.sh

```diff
@@ -24,7 +29,7 @@ ${SCRIPT_DIR}/cluster.neo4j.sh maintenance on
 # database backup
 kubectl --kubeconfig=${KUBECONFIG} -n default exec -it \
     $(kubectl --kubeconfig=${KUBECONFIG} -n default get pods | grep ocelot-neo4j | awk '{ print $1 }') \
-    -- neo4j-admin dump --database=graph.db --to=/var/lib/neo4j/$BACKUP_DATE-neo4j-dump
+    -- neo4j-admin dump --database=neo4j --to=/var/lib/neo4j/$BACKUP_DATE-neo4j-dump
```

Comment:
To be discussed, because @ulfgebhardt gave the hint that the changed db name `neo4j` come by loading dumps to `sender.fm` and `yunite.me`.

That seems plausable to me!

Therefore he suggests to set the default db name in the deployment of the db and then we can remove the `--database=graph.db` option.

So inverstigate this possibility, because it may be that this is only possible since version Neo4j v5 and not on our v4.4.

Reply:
I find that you cannot change the database default name in Neo4j community edition, but only in enterprise edition:

- see https://neo4j.com/docs/operations-manual/4.4/reference/procedures/#_list_of_procedures: `dbms.cluster.setDefaultDatabase()`

Reply:
Ok this database is a very sad story ;)

If you want to solve the problem on database level(what I still recommend since it reduces code-impact) you can rename the databases on all those clusters still using `graph.db` as database name.
(`graph.db` is the neo4j 3.x database name and should not occur on any future installations)

Sadly the database does not provide a rename database command (as far as my research goes), but using dump & restore can solve this issue in no-time.

```
Backup the existing database into data/dumps folder by running this command in the terminal window:

bin/neo4j-admin backup --backup-dir=data/dumps --database=xyz

Restore this backup as a different database:

bin/neo4j-admin restore --from=data/dumps/xyz --database=xyzz --force
```
[Source](https://community.neo4j.com/t/creating-a-database-and-then-renaming-it-or-copying-it-to-another-within-the-same-instance/36374)

If you want to use the default database name you have to do the following:
1. remove all `--database=graph.db` from all commands, do not replace it with `--database=neo4j `, just remove it
2. All databases which are still imported and have the name `graph.db` -> Log into cluster and dump+restore database as described above with default name (`neo4j`).
3. ...
4. Profit

The option you suggested is still valid, just has way more implications
- make database name configurable
- configure it for every cluster accordingly
- adjust all scripts
- ensure proper env variable loading and passing through scripts
- ...

Reply:
Oh and here is also a copy command: https://neo4j.com/docs/operations-manual/4.4/backup-restore/copy-database/ lots of red text tho.

Reply:
I find the solution to set the default database by configuration by use `NEO4J_dbms_default__database` as an environment variable when starting.

Tried that out and works.

Thanks for your help @ulfgebhardt !

Reply:
I have removed `--database=`